src/view.js

```diff
@@ -206,6 +206,30 @@ const View = Backbone.View.extend({
       .value();
   }
 }, {
+
+  // Sets the templateContext for this view class including
+  // the current templateContext of its prototypes
+  setTemplateContext(context) {
+    const templateContext = {};
+    let ViewClass = this.prototype;
+
+    while (ViewClass.prototype !== Backbone.View) {
+      ViewClass = ViewClass.prototype;
+      if (ViewClass._templateContext) {
+        _.extend(templateContext, ViewClass._templateContext);
+      }
```

Comment:
oh right..
guess this should be `templateContext = _.extend({}, ViewClass._templateContext, templateContext);`